apps/editing-toolkit/editing-toolkit-plugin/full-site-editing-plugin.php

```diff
@@ -383,25 +383,15 @@ function load_tags_education() {
 
 /**
  * Help center
- * At the moment we're disabling the help center.
  */
 function load_help_center() {
-	// enable help center for all proxied users.
-	$is_proxied = isset( $_SERVER['A8C_PROXIED_REQUEST'] ) ? sanitize_text_field( wp_unslash( $_SERVER['A8C_PROXIED_REQUEST'] ) ) : false || defined( 'A8C_PROXIED_REQUEST' ) && A8C_PROXIED_REQUEST;
-
 	// disable help center in P2s.
 	if ( defined( 'IS_WPCOM' ) && IS_WPCOM && \WPForTeams\is_wpforteams_site( get_current_blog_id() ) ) {
 		return false;
 	}
 
-	$current_segment = 75; // segment of existing users that will get the help center in %.
-	$user_segment    = get_current_user_id() % 100;
-
-	if ( $is_proxied || $user_segment < $current_segment ) {
-		require_once __DIR__ . '/help-center/class-help-center.php';
-	}
+	require_once __DIR__ . '/help-center/class-help-center.php';
 }
-add_action( 'plugins_loaded', __NAMESPACE__ . '\load_help_center' );
```

Comment:
I think you need this to load the Help Center plugin don't you?

Reply:
Yes ssirr